va/va.go

```diff
@@ -80,20 +94,37 @@ func NewValidationAuthorityImpl(
 		},
 		[]string{"type", "result"})
 	stats.MustRegister(validationTime)
+	remoteValidationTime := prometheus.NewHistogramVec(
```

Comment:
Now that we have more than one stat in this object, I think it's time to move these fields into a "stat" sub-object, with a separate initialization function to create and register them (to simplify the New function, and keep the field list manageable).